moveit_kinematics/CMakeLists.txt

```diff
@@ -33,21 +50,23 @@ pluginlib_export_plugin_description_file(moveit_core cached_ik_kinematics_plugin
 
 include_directories(${THIS_PACKAGE_INCLUDE_DIRS})
 
+add_subdirectory(cached_ik_kinematics_plugin)
+add_subdirectory(ikfast_kinematics_plugin)
 add_subdirectory(kdl_kinematics_plugin)
 add_subdirectory(lma_kinematics_plugin)
 add_subdirectory(srv_kinematics_plugin)
-add_subdirectory(ikfast_kinematics_plugin)
-add_subdirectory(cached_ik_kinematics_plugin)
 add_subdirectory(test)
 
-ament_export_libraries(
-    moveit_kdl_kinematics_plugin
-    moveit_lma_kinematics_plugin
-    moveit_srv_kinematics_plugin
-    moveit_cached_ik_kinematics_plugin)
-ament_export_include_directories(include)
```

Comment:
what about the include directories? Are these already exported with the install command?

Reply:
Also, I thought that we would still have to call `ament_export_libraries()`.